nin/frontend/app/scripts/directives/editor/GraphEditorInputOutput.js

```diff
@@ -77,9 +77,9 @@ class GraphEditorInputOutput extends React.Component {
         strokeWidth: 2,
         onClick: event => {this.props.editor.startOrCompleteConnection(this.props.item);},
       }),
-      this.props.scale > 1.5 ? e('text', {
+      this.props.scale >= 1.5 ? e('text', {
         x: 0,
-        y: 20,
+        y: 10 * this.props.scale,
```

Comment:
Why this change? Compare before:

![image](https://cloud.githubusercontent.com/assets/578029/23968552/72685b8c-09c3-11e7-8c8d-9f23bdfe2f8d.png)

after:

![image](https://cloud.githubusercontent.com/assets/578029/23968580/949422b8-09c3-11e7-8e54-9b5fa4c40292.png)